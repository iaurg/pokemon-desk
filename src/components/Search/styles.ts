import styled from 'styled-components'

export const Wrapper = styled.main`
  input {
    background-color: #f4f5f5;
    border: none;
    padding: 1.5rem;
    border-radius: 0.8rem;
    width: 100%;
  }
`
