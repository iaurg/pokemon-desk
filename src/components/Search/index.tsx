import * as S from './styles'

const Search = () => (
  <S.Wrapper>
    <input type="text" placeholder="Search for pokemons" />
  </S.Wrapper>
)

export default Search
