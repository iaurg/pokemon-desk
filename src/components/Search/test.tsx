import { render, screen } from '@testing-library/react'

import Search from '.'

describe('<Search />', () => {
  it('should render the heading', () => {
    const { container } = render(<Search />)

    expect(screen.getByPlaceholderText(/search/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
