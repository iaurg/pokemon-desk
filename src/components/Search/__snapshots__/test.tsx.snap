// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> should render the heading 1`] = `
.c0 input {
  background-color: #f4f5f5;
  border: none;
  padding: 1.5rem;
  border-radius: 0.8rem;
  width: 100%;
}

<main
  class="c0"
>
  <input
    placeholder="Search for pokemons"
    type="text"
  />
</main>
`
